lib/Limitation/Mapper/ParentDepthLimitationMapper.php

```diff
@@ -29,4 +32,9 @@ protected function getSelectionChoices()
 
         return $choices;
     }
+
+    public function mapLimitationValue(Limitation $limitation)
+    {
+        return $limitation->limitationValues;
```

Comment:
Minor: This is the same as for the NullLimitationMapper. We could have a `LimitationValueMapper` default implementation which implements the method in this form. But I saw no other cases, and code duplication is minimal.